#! /bin/sh /usr/share/dpatch/dpatch-run
## 20implict_pointer_conversion.dpatch by Florian Ragwitz <rafl@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Include malloc.h to avoid implict pointer conversion on 64 
## DP: architectures. See #377725.

@DPATCH@

--- openmpi-1.1.orig/ompi/mca/mpool/openib/mpool_openib_module.c	2006-08-07 15:56:18.428275656 +0200
+++ openmpi-1.1/ompi/mca/mpool/openib/mpool_openib_module.c	2006-08-07 15:55:50.757482256 +0200
@@ -26,6 +26,7 @@
 #include "ompi/mca/rcache/rcache.h"
 #include "ompi/mca/rcache/base/base.h"
 #include "ompi/mca/mpool/base/base.h"
+#include <malloc.h>
 
 extern uint32_t mca_mpool_base_page_size;
 extern uint32_t mca_mpool_base_page_size_log; 
